LICENSE

```diff
@@ -0,0 +1,2 @@
+Copyright 2018 PDFTron Systems Inc. All rights reserved.
```

Comment:
Since we're adding it new might as well make this 2024 Apryse, see here for example https://github.com/ApryseSDK/webviewer-ui/blob/10.12/LICENSE

Reply:
Updated LICENSE contents per request.